arp_scripts/submit_smd.sh

```diff
@@ -231,12 +232,17 @@ if [ -v LOGDIR ]; then
     LOGFILE=$LOGDIR'/'$LOGFILE
 fi
 
-SBATCH_ARGS="-p $QUEUE --ntasks-per-node $TASKS_PER_NODE --ntasks $CORES -o $LOGFILE --exclusive"
+#SBATCH_ARGS="-p $QUEUE --ntasks-per-node $TASKS_PER_NODE --ntasks $CORES -o $LOGFILE --exclusive"
+SBATCH_ARGS="-p $QUEUE --ntasks-per-node $TASKS_PER_NODE --ntasks $CORES -o $LOGFILE" # test without exclusive for now
```

Comment:
Exclusive was there so that we would keep all the IO bandwidth & memory for our job. I suspect memory may not be an issue as we (now?) request only total mem/number of available cores on the machine by default.

Reply:
Not running exclusive means that we don't have to wait for a full machine to be available. This can speed queue times significantly. Let's see if we run into any issues with that?